area4/dividers.txt

```diff
@@ -1,4 +1,4 @@
-# area4 dividers list (make sure to only put 12 characters per divider)
+﻿# area4 dividers list (make sure to only put 12 characters per divider)
```

Comment:
What happened here?

Reply:
idk why does it appear, but i will review, sorry, i dont change anything in this line.

Reply:
i'll send this again

Reply:
fixed